src/rsz/src/RecoverPower.cc

```diff
@@ -468,4 +487,36 @@ int RecoverPower::fanout(Vertex* vertex)
   return fanout;
 }
 
+void RecoverPower::printProgress(int iteration, bool force, bool end) const
+{
+  const bool start = iteration == 0;
+
+  if (start && !end) {
+    logger_->report("Iteration |  Resized |   WNS    | Endpt");
```

Comment:
I think it would be nice to have area percentage here as well (presumably negative).

Reply:
easy enough to add